### PR TITLE
Suvela_coding_exam4

### PR DESCRIPTION
**Pull request by SuvelaKoct2020**
exam4_1
exam4_2

### CODE REVIEW

**Review by harshvegada — CHANGES_REQUESTED** [review]
I don't think so you require two for loops here,
using single loop you can check,

using charAt() you can check, hope you get this hint

**Comment by SuvelaKoct2020** [comment]
Please check now. @harshvegada

File: src/suvela/exams/Exam4_2.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+input 2:
+String s1 = "pune";
+String s2 = "pune";
+
+output 2: both string are identical*/
+package suvela.exams;
+
+public class Exam4_2 {
+	void isStringIdentical(String s, String s2) {
```
Comment (harshvegada) [inline src/suvela/exams/Exam4_2.java:18]: Ideally you should this code
think and then write logic


void isStringIdentical(String s, String s2) {
		System.out.println("given strings are:" + s + "," + s2);
		boolean flag = false;
		if (s.length() == s2.length()) {
			for (int i = 0; i < s.length(); i++) {
				if (s.charAt(i) == s2.charAt(i))
					flag = true;
				else {
					flag = false;
					break;
				}
			}
		}
		if (flag)
			System.out.println("both strings are identical");
		else
			System.out.println("both strings are not identical");
	}

**Review by harshvegada — APPROVED** [review]
more practice require on coding

**Pull request merged**